app/src/main/resources/swagger/api-docs.json

```diff
@@ -1770,6 +1770,89 @@
         } ]
       }
     },
+    "/v2/tokens/aggregates-csv/{onboardingId}/products/{productId}" : {
```

Comment:
it might be better `/v2/tokens/{onboardingId}/products/{productId}/aggregates-csv` (which tokens and for what products you want the aggregates-csv)